EDO chapter corrections (#135)
daffidwilde/thesis: chapters/edo/main.tex

        for each \(j = 1, \ldots, |\mathcal{P}|\). That is, \(C\) defines the
        minimum and maximum number of columns a dataset may have from each
        distribution in \(\mathcal{P}\).
\end{itemize}

The remaining two parameters are optional, but can be useful for introducing and
focusing exploration, respectively:

\begin{itemize}
    \item A second selection parameter, \(l \in [0, 1]\), to allow for a
        small proportion of `lucky' individuals to be carried forward.
    \item A shrink factor, \(s \in [0, 1]\), defining the relative size of a
        component of the search space to be retained after adjustment.
\end{itemize}

> Choosing an optimal set of parameters for any given run of EDO is not concrete

Reviewer: ```suggestion
Choosing an optimal set of parameters for any given run of EDO is not an objective exercise
```

use. This case study uses the proposed method to reproduce some known results
about the clustering of data in the absence of any external forces and examines
how clustering algorithms are typically evaluated. In particular, the focus will
be on the well-known \(k\)-means (Lloyd's) algorithm.

Clustering has been chosen here as it is a well-understood problem that is
accessible --- most notably when restricted to two dimensions. However, the
overall approach taken in this case study is generic and would apply to any
machine learning method. First, choose an algorithm (or set of algorithms) and
set the initial fitness function to be a metric of interest for that method.

> The remainder of the process is an iterative loop of studying the generated

Reviewer: ```suggestion
The remainder of the process is a cycle of studying the generated
```

I think this reads a tad better.
